study.md

```diff
@@ -23,4 +23,15 @@ In your own words, describe why someone would use Ember.
 
 ```md
 <!-- your answer here -->
+Ember is a front end framework.  Someone would want to use Ember, or any other frontend framework,
+  for many reasons.  One being it helps manage app complexity.  It keeps your app seperated into
+  standardized, manangeable files.  It allows there to be structure and consistency, allowing for large apps
+  to be maintained for years.
+Ember provides a model, view, controller pattern on the front end.  There basic parts of Ember are:
+  Routes, each part of the URL has a route object that controls what the user is seeing.
+  Models, with each route comes a model.  The model is responsible for the data associated with the current view of the
+  application.  Json objects can be loaded from a server and use them, or Ember data can serve this purpose.
+  Templates are used to build the HTML.  They are written in a version of handlebars.
+  Components are custom HTML tags.  The behavior is implemented with Javascript, and the display is contolled by handlebars
```

Comment:
Ermm... not quite. Components are a combination of a template and an object for holding data.behavior.